.github/workflows/php-stan.yaml

```diff
@@ -55,6 +60,17 @@ jobs:
       - name: Install composer dependencies
         uses: ramsey/composer-install@v2
 
+      - name: Cache PHPStan Results
+        uses: actions/cache@v3
+        with:
+          path: ${{ inputs.cache_path }}
+          key: "${{ runner.os }}-phpstan-${{ github.run_id }}" # always unique key - always writes a new cache
+          restore-keys: |
+            ${{ runner.os }}-phpstan-
+
+      - name: Show Files
+        run: ls -la .
```

Comment:
We probably don't need this long term, but I'm guessing this is here for validation.

Reply:
Since we have the `-vvv` I'll remove it, thanks!